primitives/core/src/lib.rs

```diff
@@ -70,6 +70,8 @@ mod changes_trie;
 #[cfg(feature = "std")]
 pub mod traits;
 pub mod testing;
+#[cfg(feature = "std")]
+pub mod tasks;
```

Comment:
I'm not 100% sure this fits into `sp-core`, but at the same time we don't really have a better place currently, so I'm fine with this (unless someone has a better idea where to put it).

Reply:
We probably could put everything tasks related (Also the `CloneableSpawn` trait) in its own crate, but the current amount of code isn't also that much.